app/javascript/components/TripFormModal/TripFormPlaces.js

```diff
@@ -61,6 +61,7 @@ const TripFormPlaces = () => {
                     styles.error,
                   ]
                 )}
+                aria-labelledby="countriesOrStates"
```

Comment:
Instead of adding an `id` for the label and mapping it to the input with `aria-labelledby`, we can use the `inputId` prop (https://react-select.com/props#select-props) which will work with the existing `for` attribute on the label.
```
 inputId={FIELD_NAMES.COUNTRIES}
```

Reply:
See first comment.